giskard/rag/report.py

```diff
@@ -156,18 +156,18 @@ def save(self, folder_path: str):
 
         report_details = {"recommendation": self._recommendation}
         with open(path / "report_details.json", "w", encoding="utf-8") as f:
-            json.dump(report_details, f)
+            json.dump(report_details, f, ensure_ascii=False)
 
-        self._knowledge_base._knowledge_base_df.to_json(path / "knowledge_base.jsonl", orient="records", lines=True)
+        self._knowledge_base._knowledge_base_df.to_json(path / "knowledge_base.jsonl", orient="records", lines=True, force_ascii=False)
```

Comment:
making pre-commit test to pass
```suggestion
        self._knowledge_base._knowledge_base_df.to_json(
            path / "knowledge_base.jsonl", orient="records", lines=True, force_ascii=False
        )
```